index.js

```diff
@@ -169,10 +176,61 @@ module.exports = (robot) => {
     authorize(JSON.parse(clientSecret), checkClaSheet);
   };
 
+  var checkMergeConflicts = async function(context) {
+    var mergeConflictLabel = ['Has Merge Conflicts'];

```

Comment:
The label should have `PR: don't merge`  prefix.

Reply:
I think we decided to remove this prefix as a PR with a merge conflict cannot be merged so there will not be any accidental merging till the conflicts are resolved. Though I am happy to change it :)

Reply:
To keep it consistent with the other labels, we should have it :)